budgetApp1/budgetApp1/AccountViewController.swift

```diff
@@ -52,11 +54,45 @@ extension AccountViewController: UITableViewDelegate, UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard let cell = tableView.dequeueReusableCell(withIdentifier: AccountViewCell.identifier) as? AccountViewCell else { return UITableViewCell() }
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: NibIdentifiers.AccountViewCellIdentifier) as? AccountViewCell else { return UITableViewCell() }
         guard let account = viewModel.account(atIndex: indexPath.section) else { return UITableViewCell() }
         cell.populateWith(account: account)
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        let cornerRadius: CGFloat = 18.0
+        
+        let layer = CAShapeLayer()
+        let pathRef = CGMutablePath()
+        let bounds = cell.bounds.insetBy(dx: 7, dy: 0)
+        self.tableView.separatorStyle = .none
+        if indexPath.row == 0 && indexPath.row == tableView.numberOfRows(inSection: indexPath.section) - 1 {
+            pathRef.addRoundedRect(in: bounds, cornerWidth: cornerRadius, cornerHeight: cornerRadius)
+        } else if indexPath.row == 0 {
+            pathRef.move(to: CGPoint(x: bounds.minX, y: bounds.maxY))
+            pathRef.addArc(tangent1End: CGPoint(x: bounds.minX, y: bounds.minY), tangent2End: CGPoint(x: bounds.midX, y: bounds.midY), radius: cornerRadius)
```

Comment:
all of this logic should be in the cell

Reply:
May I please change this later? I'be been getting a lot of errors trying to fix this? Can I do it in the next PR?

Reply:
happy for you to change it in the next PR so we can get this one through. But please change it

Reply:
Okay